packages/react/src/components/TreeView/TreeNode.tsx

```diff
@@ -151,20 +314,22 @@ const TreeNode = React.forwardRef(
        *   reduced spacing between the expand icon and the node icon + label)
        */
       const calcOffset = () => {
-        // parent node with icon
-        if (children && Icon) {
-          return depth + 1 + depth * 0.5;
-        }
-        // parent node without icon
-        if (children) {
-          return depth + 1;
-        }
-        // leaf node with icon
-        if (Icon) {
-          return depth + 2 + depth * 0.5;
+        if (depth != null) {
```

Comment:
I don't think this is necessary since you updated the default function parameter to always set it to `0`?

Reply:
Here I'm performing the extra check because typescript detects it as possibly 'undefined'
![image](https://github.com/carbon-design-system/carbon/assets/25573926/4dbc7ba1-988b-41c0-bc71-a611c1ed060d)

Reply:
Ah weird, yeah I see on second glance it's actually not the default function parameter that you updated, it's just the `depth: (depth ?? 0)` in the `cloneElement` call. This looks good to me then 👍